docs/sphinx/sst/running_simeng_with_sst.rst

```diff
@@ -0,0 +1,361 @@
+one SubComponent is being loaded into the specified slot. Each SubComponent must be loaded into a unique ``slot_index`` and some SubComponents will require the 
+indexes to be incremental.
+
+.. code-block:: python
+
+    import sst
+    # Memory controller from memHierarchy library
+    memctrl = sst.Component("memory", "memHierarchy.MemController")
+    memctrl.addParams({
+        ...
+    })
+    # Memory controller has a subcomponent slot for the memory backend.
+    memory = memctrl.setSubComponent("backend", "memHierarchy.simpleMem")
+    memory.addParams({
+        ...
+    })
+
+SimEng SST Configuration
+------------------------
+
+Configuring the SimEngCore
+**************************
+As discussed earlier, the SimEng core has been wrapped in the ``SimengCoreWrapper`` class and registered as a custom SST Component. Instantiating and using this 
+component is fundamental to running the simulation. The component has the following parameters:
+
+* ``config_path``: Path to YAML configuration file needed by SimEng for configuration of the core microarchitecture under simulation.
+* ``executable_path``: Path to the executable to run inside SimEng.
+* ``executable_args``: Arguments provided to the executable.
+* ``clock``: The frequency of clock ticking the SimEng Core e.g. 1GHz (S.I units accepted).
+* ``max_addr_range``: maximum address which can be accessed by SimEng.
+* ``cache_line_width``: Width of the cache line (in bytes).
+
+Configuring StandardInterface
+*****************************
+After the ``simengcore`` has been instantiated, the ``StandardInterface`` has to be set into the ``memory`` slot of simengcore.
+
+.. code-block:: python
+
+   iface = cpu.setSubComponent("memory", "memHierarchy.standardInterface")
+
+Configuring the Cache
+*********************
+Next, the L1 cache needs to be configured. All configuration parameters (and their documentation), ports and slots for the cache can be found using the 
+command ``sst-info memHierarchy.Cache``. The default cache parameters provided by the installation are defined below:
+
+.. code-block:: python
+
+   l1cache = sst.Component("l1cache.mesi", "memHierarchy.Cache")
+   l1cache.addParams({
+        "access_latency_cycles" : "2",
+        "cache_frequency" : "2Ghz",
+        "replacement_policy" : "nmru",
+        "coherence_protocol" : "MESI",
+        "associativity" : "4",
+        "cache_line_size" : "64",
+        "debug" : 1,
+        "debug_level" : 1,
+        "L1" : "1",
+        "cache_size" : "200KiB",
+    })
+
+Configuring the MemoryController and MemBackend
+***********************************************
+
+As discussed above, the ``MemoryController`` and the ``MemBackend`` are the last levels in the memory hierarchy. The ``MemBackend`` is a subcomponent of the 
+``MemoryController`` (``slot: backend``) and for the SimEng simulation currently, the ``SimpleMem`` backend is used. Other backends are also supported.
```

Comment:
*"...``MemoryController`` (``slot: backend``) and for the SimEng simulation the ``SimpleMem`` backend is used. Other backends are also supported."